pkg/gitprovider/azure-devops.go

```diff
@@ -63,7 +63,7 @@ func (g *AzureDevOpsGitProvider) GetNamespaces() ([]*GitNamespace, error) {
 	return nil, err
 }
 
-func (g *AzureDevOpsGitProvider) GetRepositories(namespace string) ([]*GitRepository, error) {
+func (g *AzureDevOpsGitProvider) GetRepositories(namespace string, page, perPage int) ([]*GitRepository, error) {
```

Comment:
Please define the args with a struct called `ListOptions` that will be reused with all list methods.

Reply:
done